validator_instance.go

```diff
@@ -271,6 +272,33 @@ func (v *Validate) RegisterStructValidationCtx(fn StructLevelFuncCtx, types ...i
 	}
 }
 
+// RegisterStructValidationMapRules registers validate map rules
```

Comment:
Can you add some addition information regarding that rules override any tags at the struct level?

Reply:
Ah I can add it.